app/main.py

```diff
@@ -35,6 +35,30 @@ async def root():
     }
 
 
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+def verify_password(plain_password, hashed_password):
+    return pwd_context.verify(plain_password, hashed_password)
+
+def get_password_hash(password):
+    return pwd_context.hash(password)
+
+def authenticate_user(collection_users, citizen_id: str, password: str):
+    user = collection_users.find_one({"_id": citizen_id}, {"cvv": 1})
+    if not user:
+        return False
+    if not verify_password(password, user["cvv"]):
+        return False
+    return user
+
+@app.post("/api/v1/validate/{citizen_id}/{cvv}")

```

Comment:
- this method should be get
- cvv should give it in request body or other method but not in parameter
```suggestion
@app.post("/{parameter}")
def simple(parameter: BaseModel, permission: bool = Depends(verify_password)):
```
this is an example of code that pass in body not in parameter